frontend/pages/hosts/details/DeviceUserPage/components/SettingUpYourDevice/SetupSoftwareStatusTable/SetupSoftwareStatusTable.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+import { ISetupSoftwareStatus } from "interfaces/software";
+
+import TableContainer from "components/TableContainer";
+import EmptyTable from "components/EmptyTable";
+
+import generateColumnConfigs from "./SetupSoftwareStatusTableConfig";
+
+const baseClass = "setup-software-status-table";
+
+interface ISetupSoftwareStatusTableProps {
+  statuses: ISetupSoftwareStatus[];
+}
+
+const SetupSoftwareStatusTable = ({
+  statuses,
+}: ISetupSoftwareStatusTableProps) => {
+  const columnConfigs = generateColumnConfigs();
+
+  return (
+    <div className={baseClass}>
+      <TableContainer
+        columnConfigs={columnConfigs}
+        data={statuses}
+        isLoading={false}
+        showMarkAllPages={false}
+        isAllPagesSelected={false}
+        disableTableHeader={false}
+        disablePagination
+        pageSize={statuses.length}
```

Comment:
i'm assuming this page scrolls?

Reply:
Yea, figured so since pagination was never mentioned